letsencrypt-auto

```diff
@@ -99,5 +110,9 @@ fi
 
 # Explain what's about to happen, for the benefit of those getting sudo
 # password prompts...
-echo "Running with virtualenv:" $SUDO $VENV_BIN/letsencrypt "$@"
-$SUDO $VENV_BIN/letsencrypt "$@"
+if [ -z "$SCL" ] ; then
+    echo "Running with virtualenv:" $SUDO $VENV_BIN/letsencrypt "$@"
+    $SUDO $VENV_BIN/letsencrypt "$@"
+else
+    echo "Now run: $SUDO $SCL \"$VENV_BIN/letsencrypt $@\""
```

Comment:
Yeah. I agree, however I couldn't figure it out.

My original go had `$SUDO $SCL $VENV_BIN/letsencrypt "$@"` (i.e. without the echo) but it fails. I tried escaping the quotes but to no avail. `$SCL` sets up the correct python ENV variables but something goes amiss when called in the script with SUDO... works fine if the user copy/paste.

Any idea on what I'm missing?